feeds.go

```diff
@@ -131,10 +131,10 @@ func (ft *FeedTag) Next() bool {
 		},
 	)
 	if err == nil {
-		newFT := FeedTag{}
-		err = json.Unmarshal(body, &newFT)
+		newFT := &FeedTag{}
+		err = json.Unmarshal(body, newFT)
 		if err == nil {
-			*ft = newFT
+			*ft = *newFT
```

Comment:
I have a question. What's the difference between `*ft = *newFT` and `ft = newFT`? Both of them are a pointer of FeedTag. So I think you don't need `*` for it.

Reply:
Yes both of them are pointer, they have the same type but we need *ft to update inner value of feedTag.
Please view attached playground https://play.golang.org/p/jVG7TjccB8W